partitura/score.py

```diff
@@ -2444,10 +2448,17 @@ def end_note(self, note):
             note.tuplet_stops.append(self)
         self._end_note = note
 
+    @property
+    def duration_multipler(self) -> Fraction:
+        return Fraction(self.normal_notes, self.actual_notes)
```

Comment:
I think there is a typo in this property. Also, is it used anywhere?

Reply:
Yes, I'll fix the typo. It is not used anywhere so far, it's a utility inspired by the [`tupletMultiplier`](https://www.music21.org/music21docs/moduleReference/moduleDuration.html#music21.duration.Tuplet.tupletMultiplier) property of music21 and I thought that it might be useful, however I can remove it if you think this does not belong to partitura

Reply:
No, it could be nice to have it. Maybe add some documentation on the property and a link to the music21 property so if somebody wants to use it in the future it would be easier to know how.

Reply:
I'll do that!